mllib/src/test/scala/org/apache/spark/ml/MLEventsSuite.scala

```diff
@@ -239,6 +239,7 @@ class MLEventsSuite
       events.map(JsonProtocol.sparkEventToJson).foreach { event =>
         assert(JsonProtocol.sparkEventFromJson(event).isInstanceOf[MLEvent])
       }
+      sc.listenerBus.waitUntilEmpty(timeoutMillis = 10000)
```

Comment:
Maybe we can add an assert for the size of `MLEvent` of `events` within `eventually` to prevent too. But I think the current change is good enough. We can change it later if we see another actual issue.